src/completers/git.zsh

```diff
@@ -152,7 +169,17 @@ _fzf_complete_git() {
                 ;;
 
             *)
-                _fzf_complete_git-ls-files '' '--multi' $@
+                if [[ ${${(Q)${(z)arguments}}[(r)--source]} = '--source' ]] || [[ ${${(Q)${(z)arguments}}[(r)-s]} = '-s' ]]; then
```

Comment:
Just check for whether the expansion results in a non-empty string. I'm not sure why there have been already two ways to do something like this written in this `git.zsh` though. Correct me if I'm wrong but this implementation looks like it doesn't accept `-Xs` where `X` is any other short option.
```suggestion
                if [[ -n ${${(Q)${(z)arguments}}[(r)--source]} ]] || [[ -n ${${(Q)${(z)arguments}}[(r)-s]} ]]; then
```